app/scripts/views/mixins/sms-mixin.js

```diff
@@ -22,11 +22,8 @@ define((require, exports, module) => {
      * @returns {String[]}
      */
     getSmsFeatures () {
-      const features = [];
-      if (this.isInExperimentGroup('sendSms', 'signinCodes')) {
```

Comment:
Like the [call to isInExperiment in the other PR](https://github.com/mozilla/fxa-content-server/pull/5687/files#diff-dd964d8b8d3c6dfce7f42f2a6308e425L186), `isInExperimentGroup` always returned false because no country is passed in. We have decided that if SMS is enabled, we always send a signinCode, so this gating logic is no longer needed.